frontend/src/action-creator.async.js

```diff
@@ -947,15 +947,41 @@ export const getNotificationList = (
   return fetchGetNotificationWall
 }
 
-export const putNotificationAsRead = (userId, eventId) => dispatch => {
+/**
+ * Put a list of notifications as read
+ * @param {String} userId
+ * @param {int[]} notificationIdList
+ * @returns
+ */
+export const putNotificationListAsRead = (userId, notificationIdList) => dispatch => {
+  return fetchWrapper({
+    url: `${FETCH_CONFIG.apiUrl}/users/${userId}/messages/read` +
+      `?notification_ids=${notificationIdList.join(',')}`,
```

Comment:
```suggestion
    url: `${FETCH_CONFIG.apiUrl}/users/${userId}/messages/read?notification_ids=${notificationIdList.join(',')}`,
```

Reply:
Just a suggestion, don't need to apply

Reply:
I've splited the line here to keep the line under 100 characters while easily readable

Reply:
ok